install_config/install/advanced_install.adoc

```diff
@@ -1644,9 +1655,27 @@ be deleted when the Elasticsearch pod terminates.
 There are three options for enabling cluster logging storage when using the
 advanced install:
 
+[discrete]
+[[advanced-installation-logging-storage-dynamic]]
+===== Option A: Dynamic
+
+Use the following variable if your {product-title} environment supports
+xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamic volume provisioning] for your cloud provider:
+
+----
+[OSEv3:vars]
+
+openshift_logging_storage_kind=dynamic
```

Comment:
I think it is better to use same variables in https://docs.openshift.com/container-platform/3.7/install_config/aggregate_logging.html .  openshift_logging_storage_kind=dynamic -> openshift_logging_es_pvc_dynamic=true

@sdodson  What do you think?

Reply:
@anpingli, I think you're correct. I went ahead and updated the references. We still have "openshift_logging_storage_kind=nfs" in the NFS section, but think we should leave those as is for the time being. What do you think?

Reply:
@bmcelvee  Agree, The customer may/allow  use  NFS for test purpose.